chpl/chpl-api/e2e/script/newman-run-dev.js

```diff
@@ -80,6 +90,7 @@ const jobs = [
     cb => newman.run(developerControllerDataTests,cb),
     cb => newman.run(developerControllerNoDataTests,cb),
     cb => newman.run(productsControllerTests,cb),
+	cb => newman.run(productVersionControllerTests,cb),
```

Comment:
This should be 4 spaces and not tab as per our formatter. Same with other scripts